auth_brute_force/models/res_authentication_attempt.py

```diff
@@ -159,6 +185,17 @@ def action_whitelist_add(self):
             ",".join(whitelist),
         )
 
+    @api.multi
+    def action_unbanned(self):
+        self.ensure_one()
+        if self.banned:
+            self.create({
+                'remote': self.remote,
+                'login': self.login,
+                'result': 'unbanned',
```

Comment:
IMHO it's better to remove this new state, because:

- Duplicated. We already have the "whitelisted" button. It's confusing the difference between "unbanned" and "whitelisted".
- It's creating an authentication attempt where there was none, so _it's actually lying_. If an attempt was banned, its history must say it was banned, no matter if later the remote is whitelisted. A new attempt must happen for it to pass, and in such case, it must be now "successful".

Reply:
Unbanned is used in order to clean your history. Then, the counters are set to 0.
If you are whitelisted, you can use brute force in order to attempt access, because you will always be trusted.